music_rnn/src/model.ts

```diff
@@ -52,6 +52,8 @@ export class MusicRNN<A extends magenta.controls.ControlSignalUserArgs> {
    * `NoteSequence` and `Tensor` objects. If not provided, a `converter.json`
```

Comment:
Please add (Optional) here too for consistency.

Reply:
Done.